charts/internal/machineclass/values.yaml

```diff
@@ -23,6 +23,9 @@ machineClasses:
 #    image: projects/coreos-cloud/global/images/coreos-stable-1576-5-0-v20180105
 #    labels:
 #      name: my-disk
+  gpu:
+    acceleratorType: nvidia-tesla-t4
+    count: 1
```

Comment:
since the default is not to have any gpu, I would suggest you comment this section.